docs/openapi/docs.go

```diff
@@ -1601,7 +1587,7 @@ var doc = `{
                     }
                 },
                 "BundleConfig": {
-                    "$ref": "#/definitions/cart.BundleConfiguration"
```

Comment:
This looks unexpected. Could you please double-check that this is wanted?

Reply:
this one correct